### ISSUE TITLE
Does not start on Ubuntu 19.10

### ISSUE THREAD

**Issue by xtermi2** [Doccrazy/stash-electron#1]
After installation via `apt` (1.2.0), as described in the README.md I get the follow error when starting the application:
```
Unhandled Error
Error: libcrypto.so.1.0.0: Kann die Shared-Object-Datei nicht Ã¶ffnen: Datei oder Verzeichnis nicht gefunden
    at Object.../build/Release/nodegit.node (webpack:///external "../node_modules/keytar/build/Release/keytar.node":1:1)
    at n (webpack:///webpack/bootstrap:9:1)
```
I get the same error with version 1.2.1pre-7 and when I manually download `stash-electron-1.2.0-x86_64.AppImage`. But `stash-electron-1.1.0-x86_64.AppImage` works fine.

![screenshot_2019-11-06 16-55-01](https://user-images.githubusercontent.com/1821973/68314513-90b43b00-00b6-11ea-9a09-9920a92948a6.png)

**Comment by Doccrazy** [comment]
You are correct, there appears to be a dependency missing required by the Nodegit version in 1.2.0. Ubuntu updated to libssl 1.1, but Nodegit requires libssl 1.0. You can try to symlink the new version, however I did not test this:
```
sudo ln -s /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 /usr/lib/x86_64-linux-gnu/libcrypto.so.1.0.0
```
The permanent solution will be to update the Nodegit dependency in Stash. Unfortunately, this introduces some breaking changes, so the update is not trivial.

**Comment by xtermi2** [comment]
Thx for the fast reply, but the sim-link solution just leads to another error. First I had to do the same for `libssl.so`, then the error is:
```
Unhandled Error
Error: /lib/x86_64-linux-gnu/libcrypto.so.1.0.0: version `OPENSSL_1.0.1' not found (required by /tmp/.org.chromium.Chromium.gkeTPq)
    at Object.../build/Release/nodegit.node (webpack:///external "../node_modules/keytar/build/Release/keytar.node":1:1)
    at n (webpack:///webpack/bootstrap:9:1)
```

**Comment by Doccrazy** [comment]
Not the cleanest solution, but I was able to fix the error by installing the libssl1.0.0 package from Ubuntu 18.04. Try on your own risk :wink: 
```
wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl1.0/libssl1.0.0_1.0.2n-1ubuntu5_amd64.deb
sudo dpkg -i libssl1.0.0_1.0.2n-1ubuntu5_amd64.deb
```

**Comment by xtermi2** [comment]
thx, this fixed the last error :+1:

**Comment by Doccrazy** [comment]
Should be fixed with Release 1.2.1
